examples/04_vector_field/01_3d_vector_field.py

```diff
@@ -16,8 +16,8 @@
 
 ###############################################################################
 # create a uniform grid with PyVista
-dim, spacing, origin = (40, 30, 10), (1, 1, 1), (-10, 0, 0)
-mesh = pv.UniformGrid(dim, spacing, origin)
+dims, spacing, origin = (40, 30, 10), (1, 1, 1), (-10, 0, 0)
+mesh = pv.UniformGrid(dims=dims, spacing=spacing, origin=origin)
```

Comment:
Why so ugly? Are you afraid that the ordering of the arguments might change? - If that is the case, I would argue that it would be similar likely that the keywords themselves could change.

Reply:
This is why ReadTheDocs failed. I just fixed it to work. And yes, this was the given solution from the docs of pyvista (from v0.33 on):
> Changed in version 0.33.0: First argument must now be either a path or vtk.vtkImageData. Use keyword arguments to specify the dimensions, spacing, and origin of the uniform grid.

See: https://docs.pyvista.org/api/core/_autosummary/pyvista.UniformGrid.html?highlight=uniformgrid#pyvista.UniformGrid